src/services/PicklistService.ts

```diff
@@ -1,10 +1,11 @@
 import api from '@/api';
 
-const getPicklists = async (): Promise<any> => {
+const getPicklists = async (payload: any): Promise<any> => {
   return api({
-    url: 'picklists', 
-    method: 'get'
-  });
+    url: '/performFind',
+    method: 'post',
+    data: payload
+  })
 }
 
 const getPicklist = async (query: any): Promise<any> => {
```

Comment:
I think we should also use performFind here.

Reply:
The performFind API does not provide pickingItemList, hence, on discussion, it is concluded that this will be done later.